build.proj

```diff
@@ -77,6 +78,12 @@
   <UsingTask TaskName="ValidateStrongNameSignatureTask" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   <UsingTask TaskName="FilterOutAutoRestLibraries" AssemblyFile="$(LibraryToolsFolder)\Microsoft.WindowsAzure.Build.Tasks.dll"  />
   
+  <!-- Building ClientRuntime projects -->
+  <Target Name="BuildClientRuntime">
+      <Exec Command="dotnet restore" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
+      <Exec Command="dotnet build --configuration $(Configuration)" WorkingDirectory="%(ClientRuntimeProjects.RootDir)\%(ClientRuntimeProjects.Directory)"/>
```

Comment:
Using MSBuild to build dotnet-cli just ends up calling the handler for `xproj`, which does the same thing, except the `.xproj` handler tries to handle the dependencies with the grace and skill of a three-legged hippopotamus. (that and the rest of the script handles it this way anyway.)